### PR TITLE
pages.es/common/*: add Spanish aliases

### PR DESCRIPTION
**Pull request by ikks**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/common/lzcmp.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# lzcmp
+
+> Este comando es un alias de `xzcmp`.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/lzcmp.md:5]:
```suggestion
- Vea la documentación para el comando original:
```

File: pages.es/common/lzegrep.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# lzegrep
+
+> Este comando es un alias de `xzgrep --extended-regexp`.
+> Vea también: `egrep`.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/lzegrep.md:6]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/lzfgrep.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# lzfgrep
+
+> Este comando es un alias de `xzgrep --fixed-strings`.
+> Vea también: `fgrep`.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/lzfgrep.md:6]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/lzgrep.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# lzgrep
+
+> Este comando es un alias de `xzgrep`.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/lzgrep.md:5]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/lzless.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# lzless
+
+> Este comando es un alias de `xzless`.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/lzless.md:5]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/mogrify.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# mogrify
+
+> Este comando es un alias de `magick mogrify`.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/mogrify.md:5]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/montage.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# montage
+
+> Este comando es un alias de `magick montage`.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/montage.md:5]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/netcat.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# netcat
+
+> Este comando es un alias de `nc`.
+> Más información: <https://manned.org/nc>.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/netcat.md:6]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/opera.md [hunk 9]
```diff
@@ -0,0 +1,8 @@
+# opera
+
+> Este comando es un alias de `chromium`.
+> Más información: <https://opera.com>.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/opera.md:6]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/trash-cli.md [hunk 10]
```diff
@@ -0,0 +1,8 @@
+# trash-cli
+
+> Este comando es un alias de `trash`.
+> Más información: <https://github.com/andreafrancia/trash-cli>.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/trash-cli.md:6]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/vivaldi.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+# vivaldi
+
+> Este comando es un alias de `chromium`.
+> Más información: <https://vivaldi.com>.
+
+- Ver documentación para el comando original:
```
Comment (kant) [inline pages.es/common/vivaldi.md:6]:
```suggestion
- Vea la documentación del comando original:
```

File: pages.es/common/lzcmp.md (hunk 1 above)
Comment (ikks) [reply pages.es/common/lzcmp.md]: Addressed, thank you :)

File: pages.es/common/lzfgrep.md (hunk 3 above)
Comment (ikks) [reply pages.es/common/lzfgrep.md]: Addressed, thank you :)

File: pages.es/common/lzegrep.md (hunk 2 above)
Comment (ikks) [reply pages.es/common/lzegrep.md]: Addressed, thank you :)

File: pages.es/common/lzfgrep.md (hunk 3 above)
Comment (ikks) [reply pages.es/common/lzfgrep.md]: Addressed, thank you :)

File: pages.es/common/lzgrep.md (hunk 4 above)
Comment (ikks) [reply pages.es/common/lzgrep.md]: Addressed, thank you :)

File: pages.es/common/lzless.md (hunk 5 above)
Comment (ikks) [reply pages.es/common/lzless.md]: Addressed, thank you :)

File: pages.es/common/lzmore.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
+# lzmore
+
+> Este comando es un alias de `xzmore`.
+
+- Ver documentación para el comando original:
```
[the remark this answers is not in the document]
Comment (ikks) [reply pages.es/common/lzmore.md]: Addressed, thank you :)

File: pages.es/common/mogrify.md (hunk 6 above)
Comment (ikks) [reply pages.es/common/mogrify.md]: Addressed, thank you :)

File: pages.es/common/montage.md (hunk 7 above)
Comment (ikks) [reply pages.es/common/montage.md]: Addressed, thank you :)

File: pages.es/common/netcat.md (hunk 8 above)
Comment (ikks) [reply pages.es/common/netcat.md]: Addressed, thank you :)

File: pages.es/common/opera.md (hunk 9 above)
Comment (ikks) [reply pages.es/common/opera.md]: Addressed, thank you :)

File: pages.es/common/trash-cli.md (hunk 10 above)
Comment (ikks) [reply pages.es/common/trash-cli.md]: Addressed, thank you :)

File: pages.es/common/vivaldi.md (hunk 11 above)
Comment (ikks) [reply pages.es/common/vivaldi.md]: Addressed, thank you :)

**Comment by ikks** [comment]
> There is inconsistency between `- Ver documentación para el comando original:` and `- Muestra la documentación del comando original:`. Could you please fix this?

Sorry, I made sure we are with the proper one now. Thank you.

**Comment by ikks** [comment]
@kant sorry about the reprocess here, all your suggestions were applied.  All the aliases [follow the template](https://github.com/tldr-pages/tldr/blob/4d5dd59133e3a3b869e765acda7d874fafd4044b/contributing-guides/translation-templates/alias-pages.md?plain=1#L161) now

**Review by kbdharun — APPROVED** [review]

**Pull request closed without merging**